en/docs/learn/rate-limiting/graphql-api/query-complexity-limitation.md

```diff
@@ -1,30 +1,29 @@
-Often, limiting the depth of a query only is not sufficient to protect a GraphQL service from complex queries. 
-This is due to some fields in a GraphQL schema being more costly to compute than others. WSO2 API-Manager introduces 
+Often, limiting only the depth of a query is not sufficient to protect a GraphQL service from complex queries. 
+The reason for this is that some fields in a GraphQL schema being more costly to compute than others. WSO2 API-Manager introduces 
 **“Query Complexity Limitation”** to address such cases. 
 
 With this strategy, a request allowed or rejected based on the complexity of the query, and the configured max complexity 
 value of the subscription policy for the corresponding API.
 
 Here we introduced the **complexity values for each of the Fields** in the schema. That **describes the 
-computation cost of resolving the particular field**. 
+the computation cost of resolving the particular field**. 
 
 <html>
 <div class="admonition note">
 <p class="admonition-title">Note</p>
 <p>
 If the field needs to call an expensive service to resolve itself, 
-then the **complexity should be at a high level** but if the field is easy to resolve and not an expensive operation, 
+then the complexity should be at a high level but, if the field is easy to resolve and not a cheap operation,  
 the **complexity should be at a low level**. If **no complexity defined** for a field, **by default it will get a value of 1.** 
 </p>
 </div> 
 </html>
 
-In run time, requested query complexity value will be calculated and blocked it exceeds the max complexity.
+In the run time, the requested query complexity value will be calculated and, blocked if the value exceeds the max complexity.
```

Comment:
This should read as,

`At runtime, a query will be blocked if its calculated complexity exceeds the specified max complexity.`